jenkins/bootstrap_test.py

```diff
@@ -1281,7 +1281,9 @@ def Check(job, name):
 
     def testBootstrapCIYaml(self):
         # TODO(krzyzacy): temp until more jobs to be converted
-        whitelist = []
+        whitelist = [
+            'kubernetes-e2e-g.e-latest-upgrade-'
```

Comment:
oh actually I bet you can just remove it since all the upgrade jobs are already converted :-)